api/integrations/webhook/webhook.py

```diff
@@ -22,9 +22,10 @@ def __init__(self, config: WebhookConfiguration):
 
     def _identify_user(self, data: typing.Mapping) -> None:
         response = call_integration_webhook(self.config, data)
```

Comment:
Under what circumstances does `call_integration_webhook` return `None` ?

Reply:
On error: https://github.com/Flagsmith/flagsmith/blob/main/api/tests/unit/webhooks/test_unit_webhooks.py#L324

Reply:
Ah ok, interesting. This solution seems fine for now I guess, but perhaps my original solution was wrong, and we should just try/except the `call_integration_webhook` function 🤔